stage1/usr_from_coreos/manifest.d/systemd

```diff
@@ -1,6 +1,7 @@
 bin/coredumpctl
 bin/journalctl
 bin/systemctl
+bin/mount
```

Comment:
Well, `kvm` flavor is based on `coreos` flavor. Currently there is no way to specify an additional manifest.